tokio-signal/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
Since the tokio project is only licensed under MIT, we might want to remove any mention of the Apache option. Maybe @carllerche can comment more here

Reply:
Yes, lets stick with MIT only for simplicity.